### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by jatinkumar762** [jatinkumar762/AI_Articles#15]
## TL;DR
How to Accelerate Learning of Deep Neural Networks With Batch Normalization
### Article Link
[Link](https://machinelearningmastery.com/how-to-accelerate-learning-of-deep-neural-networks-with-batch-normalization/)
